subsys/mgmt/mcumgr/transport/src/smp_dummy.c

```diff
@@ -10,16 +10,16 @@
  */
 
 /* Define required for uart_mcumgr.h functionality reuse */
-#define CONFIG_UART_MCUMGR_RX_BUF_SIZE CONFIG_MCUMGR_TRANSPORT_DUMMY_RX_BUF_SIZE
+#define CONFIG_MCUMGR_TRANSPORT_UART_RX_BUF_SIZE CONFIG_MCUMGR_TRANSPORT_DUMMY_RX_BUF_SIZE
```

Comment:
RFC: I'm also inclined to remove the uart/serial dependencies so that smp_dummy is completely standalone, at the expense of some code duplication, any idea?